tests/forms_tests/tests/tests.py

```diff
@@ -73,6 +73,11 @@ def test_queryset_manager(self):
         choice = ChoiceOptionModel.objects.create(name="choice 1")
         self.assertEqual(list(f.choices), [('', '---------'), (choice.pk, str(choice))])
 
+    def test_queryset_manager_has_length(self):
```

Comment:
I would put this in `tests/model_forms/tests.py` `ModelChoiceFieldTests` for now. (We could move these existing tests separately, although I think I'd combine both classes in a separate file `tests/model_forms/test_modelchoicefield.py`.)